### PR TITLE
add wand

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

File: recipes/wand/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - memory_profiler >=0.27
+    - psutil >=1.0.1
+    - sphinx >=1.0
+
+test:
+  imports:
+    - wand
+
+about:
+  home: https://github.com/chrisdev/django-wagtail-feeds
```
Comment (synapticarbors) [inline recipes/wand/meta.yaml]: This looks like the wrong `home` url.

File: recipes/wand/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pytest >=2.3.0
```
Comment (synapticarbors) [inline recipes/wand/meta.yaml]: Many of the listed `run` dependencies actually look like test dependencies:

https://github.com/dahlia/wand/blob/master/setup.py#L48-L52

File: recipes/wand/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
```
Comment (synapticarbors) [inline recipes/wand/meta.yaml]: Should `imagemagick` or `magickwand` be a requirement?

http://docs.wand-py.org/en/0.4.4/#requirements

File: recipes/wand/meta.yaml (hunk 2 above)
Comment (sannykr) [reply recipes/wand/meta.yaml]: Thanks. I have removed those dependencies.

File: recipes/wand/meta.yaml (hunk 1 above)
Comment (sannykr) [reply recipes/wand/meta.yaml]: Thanks. I have corrected that.

File: recipes/wand/meta.yaml (hunk 3 above)
Comment (sannykr) [reply recipes/wand/meta.yaml]: I think you are right. I looked at the CI details, and looks like the tests are not running.

**Pull request closed without merging**